src/test/java/org/sagebionetworks/bridge/reporter/helper/BridgeHelperTest.java

```diff
@@ -190,10 +197,67 @@ public void testSaveReportForStudy() throws Exception {
     }
     
     @Test
-    public void testGetParticipantsForStudy() throws Exception {
-        ForWorkersApi mockWorkerClient = mock(ForWorkersApi.class);
+    public void testGetStudyPartcipant() throws Exception {
+        when(mockClientManager.getClient(ForWorkersApi.class)).thenReturn(mockWorkerClient);
+        
+        StudyParticipant mockStudyParticipant = mockCallForParticipant(mockWorkerClient, USER_ID_1);
+        
+        StudyParticipant studyParticipant = bridgeHelper.getStudyPartcipant(TEST_STUDY_ID, USER_ID_1);
+        System.out.println(studyParticipant.getId());
+        assertEquals(studyParticipant.getId(), mockStudyParticipant.getId());
+        verify(mockWorkerClient).getParticipantByIdForStudy(TEST_STUDY_ID, USER_ID_1, false);
+    }
+    
+    @Test
+    public void testGetRequestInfoForParticipant() throws Exception {
+        when(mockClientManager.getClient(ForWorkersApi.class)).thenReturn(mockWorkerClient);
+        RequestInfo mockRequestInfo = mockCallForRequestInfo(mockWorkerClient, USER_ID_1);
+        
+        RequestInfo requestInfo = bridgeHelper.getRequestInfoForParticipant(TEST_STUDY_ID, USER_ID_1);
+        
+        assertSame(requestInfo.getUserId(), mockRequestInfo.getUserId());
+        verify(mockWorkerClient).getRequestInfoForWorker(TEST_STUDY_ID, USER_ID_1);
+    }
+    
+    @Test
+    public void testGetActivityEventForParticipant() throws Exception {
+        when(mockClientManager.getClient(ForWorkersApi.class)).thenReturn(mockWorkerClient);
+        ActivityEventList mockActiviyEventList = mockCallForActivityEventList(mockWorkerClient, USER_ID_1);
+        
+        ActivityEventList activityEventList = bridgeHelper.getActivityEventForParticipant(TEST_STUDY_ID, USER_ID_1);
         
-        ClientManager mockClientManager = mock(ClientManager.class);
+        assertNull(activityEventList.getItems());
+        assertSame(activityEventList.getItems(), mockActiviyEventList.getItems());
```

Comment:
You should call assertSame on the activityEventList itself. Otherwise, you're comparing null to null, which is trivially true.